app/code/Magento/InventorySales/Model/IsProductSalableForRequestedQtyCondition/IsSalableWithReservationsCondition.php

```diff
@@ -89,8 +89,9 @@ public function execute(string $sku, int $stockId, float $requestedQty): Product
 
         $qtyWithReservation = $stockItemData[GetStockItemDataInterface::QUANTITY] +
             $this->getReservationsQuantity->execute($sku, $stockId);
-        $qtyLeftInStock = $qtyWithReservation - $stockItemConfiguration->getMinQty() - $requestedQty;
-        $isEnoughQty = (bool)$stockItemData[GetStockItemDataInterface::IS_SALABLE] && $qtyLeftInStock >= 0;
+        $qtyLeftInStock = $qtyWithReservation - $stockItemConfiguration->getMinQty();
+        $isInStock = abs($qtyLeftInStock - $requestedQty) < 0.1;
```

Comment:
why do you use `0.1` ? 

I expect that we have to support low grained comparison value.
Thus, it's better to use `0.0001` for these purpoese